components/director/run.sh

```diff
@@ -121,12 +131,83 @@ else
 
     echo -e "${GREEN}Populate DB${NC}"
 
-    cat ${ROOT_PATH}/../schema-migrator/migrations/director/*.up.sql | \
-        docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
-    cat ${ROOT_PATH}/../schema-migrator/seeds/director/*.sql | \
-        docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
+    if [[ ${DUMP_DB} = false ]]; then
+        cat ${ROOT_PATH}/../schema-migrator/migrations/director/*.up.sql | \
```

Comment:
why not simply use `migrate` here ? or even better just call run.sh of the schema migrator

Reply:
It's definitely smarter to use `migrate` here 👍